File: 2553-separate-the-digits-in-an-array/2553-separate-the-digits-in-an-array.cpp
class Solution {
public:
    vector<int> separateDigits(vector<int>& nums) {
        vector<int>ans;
        for(auto &x:nums){
            string str=to_string(x);
            for(auto ch:str){
                ans.emplace_back(ch-'0');
            }
        }
        return ans;
    }
};

File: 2553-separate-the-digits-in-an-array/2553-separate-the-digits-in-an-array.java
class Solution {
    public int[] separateDigits(int[] nums) {
        List<Integer>ans=new ArrayList<>();
        for(int x:nums){
            String str=Integer.toString(x);
            for(int ch:str.toCharArray()){
                ans.add(ch-'0');
            }
        }

        int a[]=new int[ans.size()];

        for(int i=0;i<ans.size();i++){
            a[i]=ans.get(i);
        }

        return a;
    }
}